### ISSUE TITLE
How to restart service?

### ISSUE THREAD

**Issue by ngohuunam** [zapty/forever-service#81]
How to stop and start nodejs.js file again, in case of modify nodejs.js file?

Thks

**Comment by arvind-agarwal** [comment]
You can use the start and stop service commands, the nodejs files can be changed any number of times since everytime service starts it uses the latest script file found in the same location.

**Comment by ngohuunam** [comment]
is it
$ sudo forever-service stop test
than
$ sudo forever-service start test
?
sorry for my bad

**Comment by arvind-agarwal** [comment]
when you install the service you will get exact commands in console that time, usually it would be sudo service servicename start / stop

**Comment by ngohuunam** [comment]
Sorry I tried some times, but unfortunatly, it did't work, totally because my zero knowledge about linux, could you please teach me step by step? my service name: mynode. How I can stop and start again service to run newest nodejs file?
thanks

**Comment by yogeshwar607** [comment]
@ngohuunam , you can use following commands for your case.
sudo service mynode start
sudo service mynode stop
sudo service mynode restart
sudo service mynode status

**Comment by jm93** [comment]
yes, MUCH better documentation needed. none of this start/stop stuff is here. have to find it through google search and stackoverflow.

I too started trying with:
sudo forever-service stop test
and it wouldn't work

and found on stackoverflow this:
sudo service test start
which did, which makes no sense

(oddly enough when I tried sudo service test stop, it wouldn't stop, had to use sudo forever stopall, ugh)

this should be clearly documented here, instead have to go web hunting.

**Comment by arvind-agarwal** [comment]
@jm93 forever service is primarily designed to provision service for a given platform. The commands to control service are specific to a platform and not generic. When you provision a service using forever-service these commands are clearly displayed.

**Comment by jm93** [comment]
thanks for getting back

so I tried:
sudo forever-service install test --script server.js --start

and looks like it worked, I can get it from sudo forever list.

I'll delete my last comment if I can

**Comment by vikas0121** [comment]
Hi, i'm deploying the rocket chat application and want to use secondary MongoDB to read.
while setting the MONGOURL in the forever service
export MONGO_URL=mongodb://10.xx.xx.xxx:27017,10.xx.xx.xxx:27017/Chat?replicaSet=rs1&readPreference=nearest
not working for me..help needed
This is a Meteor.js application.

**Comment by arvind-agarwal** [comment]
What is not working exactly? Please provide exact command you are using to provision forever-service. And exact details and relevant logs related to the error.

________________________________
